src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderTable/PaymentBuilderTable.tsx

```diff
@@ -154,7 +159,7 @@ const useGetPaymentBuilderColumns = ({
       fetchCurrentBlockTime,
       finalizedTimestamp,
       hasMoreThanOneToken,
-      isLoading,
```

Comment:
On line 65 (inside the useMemo) you still have `isLoading={isLoading}` which means there's now a missing dependency in the dependency array of this useMemo call.

You'll need to either keep isLoading as a dependency or change that line to `isLoading={isDataLoading}` to fix it. I assume it's just a missed thing and the second option is what you intended!